doc-generator/generate.sh

```diff
@@ -41,9 +42,9 @@ for f in SC*; do
     *) level="Info" ;;
   esac
   if [ "$subcategory" != "" ]; then
-    patterns+=$(jq -cMn --arg patternId "$pattern_id" --arg level "$level" --arg category "$category" --arg subcategory "$subcategory" '{"patternId": $patternId, "level": $level, "category": $category, "subcategory": $subcategory}')
+    patterns+=$(jq -cMn --arg patternId "$pattern_id" --arg level "$level" --arg category "$category" --arg subcategory "$subcategory" --argjson enabled $enabled '{"patternId": $patternId, "level": $level, "category": $category, "subcategory": $subcategory, "enabled": $enabled}')
   else
-    patterns+=$(jq -cMn --arg patternId "$pattern_id" --arg level "$level" --arg category "$category" '{"patternId": $patternId, "level": $level, "category": $category}')
+    patterns+=$(jq -cMn --arg patternId "$pattern_id" --arg level "$level" --arg category "$category" --argjson enabled $enabled '{"patternId": $patternId, "level": $level, "category": $category, "enabled": $enabled}')
```

Comment:
Why `--argjson` instead of `--arg` as the other fields?

Reply:
--arg will output `"true"` or `"false"`. This will output `true` or `false`. It was just to keep consistency with the other repositories